self-hosted/install/installation-kubernetes.md

```diff
@@ -6,17 +6,30 @@ keywords: [installation, self-hosted, Kubernetes]
 ---
 
 import Skip from "versionContent/_partials/_selfhosted_cta.mdx";
+import IntegrationPrereqsSelf from "versionContent/_partials/_integration-prereqs-self-only.mdx";
+import KubernetesPrereqs from "versionContent/_partials/_kubernetes-prereqs.mdx";
+import KubernetesInstallSelf from "versionContent/_partials/_kubernetes-install-self-hosted.mdx";
 
 # Install TimescaleDB on Kubernetes
 
 TimescaleDB can be run inside Kubernetes using the TimescaleDB Docker 
-container images. In the past Timescale maintained Helm charts to 
-manage the Kubernetes deployment, but we now recommend that Kubernetes
-users rely on one of the amazing PostgreSQL Kubernetes operators to 
-simplify installation, configuration, and life-cycle.
+container images. 
 
 <Skip />
 
+## Prerequisites
+
+<IntegrationPrereqsSelf />
```

Comment:
Don't think you need to install self-hosted before you install self-hosted on kubernetes. Can you just add the "To follow the steps on this page:" above the kubernetes pre-requisites please.